src/Compilers/VisualBasic/Portable/Symbols/TypeSymbol.vb

```diff
@@ -579,10 +579,18 @@ Done:
             End Get
         End Property
 
+        Private Function ITypeSymbol_ToDisplayString(topLevelNullability As NullableAnnotation, Optional format As SymbolDisplayFormat = Nothing) As String Implements ITypeSymbol.ToDisplayString
+            Return ToDisplayString(format)
```

Comment:
>Return ToDisplayString(format) [](start = 12, length = 30)

I think we should throw an exception if the annotation is anything other than Oblivious/Unknown.

Reply:
that could make things unpleasant for users.  Imagine someone taking symbols and trying to view/convert them in a particular language.  They'd have to know the end capabilities of the final language.

Reply:
Agreed, I think this is fine to leave as is.

Reply:
Actually the underlying one will throw, but the real bug here you all missed was I forgot to pass the parameters along. ;-)

Reply:
No parameters missing here :)

Reply:
> No parameters missing here :)

Ah, right, this is VB.

> They'd have to know the end capabilities of the final language.

Curiously, we might have to do that in the IDE _anyways_ in a few cases since we're going to have to figure out how to compare these cross-language, although I'm kinda hoping those take the tact that we throw away nullability information not unlike we do other things for language specific stuff.